### ISSUE TITLE
Unable to close ghidra after looking up references to field

### ISSUE THREAD

**Issue by martmists-gh** [NationalSecurityAgency/ghidra#3303]
**Describe the bug**
After clicking references to a field, an error pops up, and closing the error opens it again (see screenshot)

**To Reproduce**
Unknown, many other fields worked just fine

**Expected behavior**
I expected it to show all references without issue, or at least let me do anything after closing the error with the information it *did* manage to look up

**Screenshots**
![Screenshot_20210802_093221](https://user-images.githubusercontent.com/16361449/127821700-31be8aa8-988c-4e85-bdeb-d1cd2fb12811.png)

**Environment (please complete the following information):**
 - OS: Arch Linux
 - Java Version: 11.0.12
 - Ghidra Version: 10.0.1
 - Ghidra Origin: Arch Linux [community] repository

**Comment by dragonmacher** [comment]
This stack trace appears to be a symptom and not the original issue.  Please provide your application log file if possible.  This file lives in `{user home}/.ghidra/.ghidra_{version}`.

**Comment by lab313ru** [comment]
The same here.

**Comment by dragonmacher** [comment]
I tried this quite a few different ways and have been unable to reproduce the issue.

**Comment by lab313ru** [comment]
Indeed, it's hard to reproduce. But my exception log was exactly the same.

**Comment by ghidra1** [comment]
Are you using a DataType Project Archive which resides on a Ghidra Server?

**Comment by lab313ru** [comment]
No, the crash in my case has happened without using any remote servers, just a local setup.

**Comment by ghidra1** [comment]
There is definitely a remote server object involved based upon the stack trace - that's where RMI comes into play.  You have multiple datatype archives open in the tool, I suspect one is remote on a Ghidra Server.  The purple icon background in the datatype manager conveys a versioned project file/archive.

**Comment by lab313ru** [comment]
Don't forget, I'm not the initial creator of this issue.

**Comment by dragonmacher** [comment]
> Don't forget, I'm not the initial creator of this issue.

In your case, the stack trace will be different, as you are not using any remote items.  If you get this to happen again, please provide the stack trace.
